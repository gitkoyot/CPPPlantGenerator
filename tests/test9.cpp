#include "test9.h"

