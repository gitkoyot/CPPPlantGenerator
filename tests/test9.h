#ifndef TEST9_H
#define TEST9_H

class BaseClass
{
public:
	BaseClass
();
	~BaseClass
();
private:
	/* data */
}; /* -----  end of class BaseClass
  ----- */



class DerivedClass : public BaseClass
{
public:
	DerivedClass();
	~DerivedClass();
private:
	/* data */
}; /* -----  end of class DerivedClass  ----- */


#endif



