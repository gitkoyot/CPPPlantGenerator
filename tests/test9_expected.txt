@startuml
class BaseClass{
+void BaseClass ( )
+void ~BaseClass ( )
}
class DerivedClass{
+void DerivedClass ( )
+void ~DerivedClass ( )
}

DerivedClass --|> BaseClass
@enduml

